shell/plugins/dashboard-store/mutations.js

```diff
@@ -134,25 +134,27 @@ export function resetStore(state, commit) {
 }
 
 export function remove(state, obj, getters) {
-  let type = normalizeType(obj.type);
-  const keyField = getters[`${ state.config.namespace }/keyFieldForType`](type);
-  const id = obj[keyField];
+  if (obj) {
```

Comment:
It breaks because there's not an object in a bulk deletion. I've created an issue to tackle this down.
https://github.com/rancher/dashboard/issues/6628
This has been already mentioned to @nwmac as he has been touching that part and I needed to merge conflict that.